app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :likes, dependent: :destroy
+
```

Comment:
👍 Good progress on your project! There's a small adjustment needed:

- [ ] **Adding `has_many` Relationship:** The `User` model should have a `has_many` relationship with `comments` as it does with `posts` and `likes`. This relationship ensures that a user can have multiple comments associated with them.

    ```ruby
    class User < ApplicationRecord
      # ... (existing code)

      has_many :comments, dependent: :destroy
    end
    ```

By adding this relationship, you'll maintain consistency in your data model and allow users to have associations with comments. Keep up the good work! 🚀🔗

Reply:
Thank you very much for your review. Have a nice day!